lib/api/get/meetings.ts

```diff
@@ -1,7 +1,9 @@
 import { RRule, RRuleSet } from 'rrule';
 import { nanoid } from 'nanoid';
 
-import { Meeting, MeetingsQuery, Timeslot } from 'lib/model';
+import { Meeting } from 'lib/model/meeting';
```

Comment:
Move these single imports below the multiple imports. I'm pretty sure your IDE will give you an ESLint error if you just open up the file (it'll tell you the same thing: to move single imports after multiple imports).